packages/ripple-nuxt-tide/lib/module.js

```diff
@@ -140,7 +140,11 @@ const nuxtTide = function (moduleOptions) {
     options: options
   })
 
-  this.addModule('@dpc-sdp/ripple-nuxt-ui')
+  if (process.env.BASIC_AUTH === '1') {
+    this.addServerMiddleware('./core/basic-auth.js')
```

Comment:
This will need the site update to 16.1 remove this middleware from their code, I think we should log all these update checklist in one place for now incase we forgot them. Here is where I record them now: https://digital-engagement.atlassian.net/wiki/spaces/SDP/pages/730005559/Release+Log+1.0.0-alpha.16.1#ReleaseLog1.0.0-alpha.16.1-Devnotes